### ISSUE TITLE
Add facebook and google oauth (backend)

### ISSUE THREAD

**Issue by jaiakash** [CCExtractor/beacon-backend#138]
Front end issue - https://github.com/CCExtractor/beacon/issues/160 
Currently, the app only accepts email sign in/signup. Having oauth would be better for the user, then user can sign in with ease.
Also there is PR for email verification, having oauth wont even need email verification. (Although that feature would be required for user sign in with their email).

**Comment by codesmith25103** [comment]
i want to work on this issue. Can you please assign me this issue?

**Comment by Mustafa-Azad025** [comment]
Hey @jaiakash ,
I Want To Work On This Issue Will You Assign This To Me ??

**Comment by sanket-1305** [comment]
i would like to work on this if no one else is working, can you please assign me?

**Comment by Sulochan-khadka** [comment]
@jaiakash Absolutely. Adding oauth specifically googleoauth2  and authorising Facebook user would be better as login credentials and then  store for further sessions . Should I raise a PR for this or will you assign this issue first.

**Comment by RunTerror** [comment]
hey @jaiakash  Please assign me this issue i want to work on this.

**Comment by Kaps61929** [comment]
@jaiakash @cfsmp3  I would love to contribute to this issue. Please assign it to me.

**Comment by harshsinghdev-11** [comment]
I want to work on this issue. Can you please assign me this issue?
